x-pack/packages/kbn-cloud-security-posture/src/utils/get_vulnerability_colors.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { euiThemeVars } from '@kbn/ui-theme';
+import type { VulnSeverity } from '@kbn/cloud-security-posture-common';
+import { VULNERABILITIES_SEVERITY } from '@kbn/cloud-security-posture-common';
+
+export const getSeverityStatusColor = (severity: VulnSeverity): string => {
+  switch (severity) {
+    case VULNERABILITIES_SEVERITY.LOW:
+      return euiThemeVars.euiColorVis0;
+    case VULNERABILITIES_SEVERITY.MEDIUM:
+      return euiThemeVars.euiColorVis5_behindText;
+    case VULNERABILITIES_SEVERITY.HIGH:
+      return euiThemeVars.euiColorVis9_behindText;
+    case VULNERABILITIES_SEVERITY.CRITICAL:
+      return euiThemeVars.euiColorDanger;
+    default:
+      return '#aaa';
```

Comment:
let's use some default color from EUI schema, it's a possible case for Wiz data as far as I can see

<img width="734" alt="Screenshot 2024-10-04 at 15 08 37" src="https://github.com/user-attachments/assets/e4af5421-1bb6-42a8-b304-8eaf00fb3e5a">